src/Yasumi/Provider/CommonHolidays.php

```diff
@@ -397,6 +397,41 @@ public function armisticeDay(
         );
     }
 
+    /**
+     * International Women's Day
+     *
+     * International Women's Day (IWD) is celebrated on March 8 every year.[3] It is a focal point in the movement for women's rights.
+     *
+     * @link https://en.wikipedia.org/wiki/International_Women%27s_Day
+     *
+     * @param int    $year     the year for which International Women's Day need to be created
+     * @param string $timezone the timezone in which International Women's Day is celebrated
+     * @param string $locale   the locale for which International Women's Day need to be displayed in.
+     * @param string $type     The type of holiday. Use the following constants: TYPE_OFFICIAL, TYPE_OBSERVANCE,
+     *                         TYPE_SEASON, TYPE_BANK or TYPE_OTHER. By default an official holiday is considered.
+     *
+     * @return \Yasumi\Holiday
+     *
+     * @throws \Yasumi\Exception\InvalidDateException
+     * @throws \Yasumi\Exception\UnknownLocaleException
+     * @throws \InvalidArgumentException
+     * @throws \Exception
+     */
+    public function internationalWomensDay(
+        int $year,
+        string $timezone,
+        string $locale,
+        string $type = Holiday::TYPE_OFFICIAL
```

Comment:
I would suggest not to use TYPE_OFFICIAL. Official holidays are typically those days that governments have decided these are non-working days. I think for this case it is better to use TYPE_OBSERVANCE, or TYPE_OTHER.

Reply:
I'd like to disagree. Your definition for TYPE_OFFICIAL is 

    Type definition for Official (i.e. National/Federal) holidays.

And International Women's Day is a official holiday in several countries (see [Wikipedia](https://en.wikipedia.org/wiki/International_Women%27s_Day#Around_the_world) )

Reply:
I was speaking more generally as I didn't have any background information yet. Now I do :)
You are correct regarding the TYPE_OFFICIAL definition. However, I was considering other countries that don't celebrate this holiday yet and they may not be considering this as an official holiday. Anyway, that can be decided in the future :)

So, please leave it as official then.